Dockerfile.import

```diff
@@ -1,8 +1,24 @@
 FROM ubuntu:xenial
 
+ARG BUILD_OSM2PGSQL
+ARG OSM2PGSQL_VERSION
+
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    curl ca-certificates osm2pgsql postgresql-client \
-    && rm -rf /var/lib/apt/lists/*
+    curl ca-certificates postgresql-client
+
+RUN if [ "$BUILD_OSM2PGSQL" = "False" ]; then apt-get install \
+    --no-install-recommends -y osm2pgsql && rm -rf /var/lib/apt/lists/*; fi
+
+RUN if [ "$BUILD_OSM2PGSQL" = "True" ]; then apt-get install \
+    --no-install-recommends -y make cmake g++ git libboost-dev \
+    libboost-system-dev libboost-filesystem-dev libexpat1-dev zlib1g-dev \
+    libbz2-dev libpq-dev libproj-dev lua5.2 liblua5.2-dev \
+    && rm -rf /var/lib/apt/lists/*; fi
+
+RUN if [ "$BUILD_OSM2PGSQL" = "True" ]; then cd / && \
+    git clone https://github.com/openstreetmap/osm2pgsql.git && \
+    cd osm2pgsql && git pull && git checkout $OSM2PGSQL_VERSION && \
```

Comment:
Don't pull since it's already got the latest, and don't checkout, specify a commitish on the clone.